src/utils/isDiscourseNode.ts

```diff
@@ -1,7 +1,10 @@
-import getDiscourseNodes from "./getDiscourseNodes";
+import getDiscourseNodes, { DiscourseNode } from "./getDiscourseNodes";
 import findDiscourseNode from "./findDiscourseNode";
 
-const isDiscourseNode = (uid?: string, nodes = getDiscourseNodes()) =>
-  !!findDiscourseNode(uid, nodes);
+const isDiscourseNode = (uid?: string, nodes = getDiscourseNodes()) => {
```

Comment:
I checked all the usages of `isDiscourseNode`, all seem okay to omit the default definitions for now.